config.ru

```diff
@@ -3,3 +3,4 @@
 
 require ::File.expand_path('config/environment', __dir__)
 run Rails.application
+Rails.application.load_server
```

Comment:
Why is this suddenly required?

Reply:
I checked again and in Rails 7, it's actually okay to not including this (I remembered in Rails 6.1 the application does not run properly if this line is not included)

Reverting back